snorkel/analysis/error_analysis.py

```diff
@@ -1,14 +1,14 @@
 from collections import defaultdict
-from typing import Any, Counter, Iterable, List, Mapping, Optional, Sequence, Tuple
+from typing import DefaultDict, Dict, List, Tuple
 
 import numpy as np
 
 from .utils import to_int_label_array
 
+Coord = Tuple[int, int]
 
-def error_buckets(
-    golds: np.ndarray, preds: np.ndarray, X: Optional[Sequence[Any]] = None
-) -> Mapping[Tuple[int, int], Any]:
+
+def error_buckets(golds: np.ndarray, preds: np.ndarray) -> Dict[Coord, List[int]]:
     """Return examples (or their indices) bucketed by gold label/pred label combination.
```

Comment:
Looks like you removed the functionality to return the examples; now it only returns indices. That's fine, and makes for clearer type checking, but let's update the docstring.